joeynmt/helpers.py

```diff
@@ -322,3 +332,15 @@ def symlink_update(target, link_name):
             os.symlink(target, link_name)
         else:
             raise e
+
+
+def latest_checkpoint_update(target: pathlib.Path,
+                             link_name: str) -> pathlib.Path:
+    link = pathlib.Path(link_name)
+    if link.is_symlink():
+        current_last = link.resolve()
+        link.unlink()
+        link.symlink_to(target)
+        return current_last
+    link.symlink_to(target)
+    return None
```

Comment:
Perhaps comment on the two different return types in a docstring, and please reflect it in the typing as well.

Reply:
Done.